### ISSUE TITLE
Shortcut hotkey not based on keyboard layout.

### ISSUE THREAD

**Issue by theherk** [ParthJadhav/Verve#15]
Shortcut is based on standard keys layout rather than the actual input given. To reproduce:

1. Set keyboard layout to colemak.
2. Open verve preferences.
3. Set Hotkey to `Cmd+Shift+F`

Now `Cmd+Shift+F` will not open Verve, but `Cmd+Shift+T` will.

---

As an aside, after installing on MB M1 Pro, I got `damaged and can't be opened` which had to be circumvented with `xattr -cr /Applications/verve.app`. Also, the app often opens lower and lower until it is off the screen and not usable, then at some point pops back up to the middle.

**Comment by ParthJadhav** [comment]
Hey @theherk , I'll be looking into the keyboard issue later on. Sorry for the delay.

I'm currently working on the Theming system. But the other issue is quite important. Can you explain a bit what the cause could be?

**Comment by ParthJadhav** [comment]
@theherk , I discovered that the issue is because the App is not signed. Using Intel Binary could help you a bit with that thing. you wouldn't need to use xattr command. Can you try again with the Intel binary?

**Comment by theherk** [comment]
No worries, I just thought I'd share what I discovered. Looking good by the way. If I download  verve_x86_64.dmg and install, this is the outcome.

<img width="672" alt="Screenshot 2022-12-29 at 19 01 36" src="https://user-images.githubusercontent.com/3605078/209991832-3058a46b-8b98-47d9-aee8-2735047522e2.png">

If I have misunderstood, or you'd like me to try something else, let me know.

**Comment by ParthJadhav** [comment]
If you right click and open, it should work.

**Comment by theherk** [comment]
I don't know how that would differ from just double clicking it, but sure I went ahead just to verify. Right-click, open yields the same result.

**Comment by ParthJadhav** [comment]
> I don't know how that would differ from just double clicking it, but sure I went ahead just to verify. Right-click, open yields the same result.

MacOS Usually adds an option to open the app if it's `Right-Clicked` and opened. 

From Apple Website:

> In the Finder  on your Mac, locate the app you want to open.
> 
> Don’t use Launchpad to do this. Launchpad doesn’t allow you to access the shortcut menu.
> 
> Control-click the app icon, then choose Open from the shortcut menu.
> 
> Click Open.
> 
> The app is saved as an exception to your security settings, and you can open it in the future by double-clicking it just as you can any registered app.

**Comment by theherk** [comment]
Yes right-click-opening from the context menu in finder is maybe different than just opening the application via the dock or spotlight, but it still seems unlikely that it differs from double clicking in finder. Nevertheless, I have done each of these combinations with the same result, so I'm unsure what's up. Sometimes when I get similar results, I can go to privacy and security in setting and allow the application, but I don't get that option here.

**Comment by ParthJadhav** [comment]
Did you try:

Control-click the app icon, then choose Open from the shortcut menu.

**Comment by theherk** [comment]
Yes. That is just opening, since control-click and right-click are the same, but for the sake of posterity I did. Same outcome.

**Comment by ParthJadhav** [comment]
That's weird. So maybe for you only `xattr -cr /Applications/verve.app` works
